backend/src/main/resources/liquibase/db-changelog-1.0.xml

```diff
@@ -261,21 +261,6 @@
         <dropIndex tableName="project" indexName="p_name_wspace_wspace_type"/>
     </changeSet>
 
-    <changeSet id="11.1.2-index-projectname-workspace-workspacetype-deleted" author="anandJ">
-        <preConditions onFail="MARK_RAN">
-            <not>
-                <indexExists indexName="p_name_wspace_wspace_type_deleted"/>
-            </not>
-        </preConditions>
-        <createIndex indexName="p_name_wspace_wspace_type_deleted"
-                     tableName="project">
-            <column name="name" type="varchar(255)"/>
-            <column name="workspace" type="varchar(255)"/>
-            <column name="workspace_type" type="int"/>
-            <column name="deleted" type="boolean"/>
-        </createIndex>
-    </changeSet>
-
```

Comment:
moved into the index changeset file

Reply:
I updated and moved the delete tag closer to the second creation